tests/migrations/test_writer.py

```diff
@@ -251,6 +251,14 @@ def test_serialize_constants(self):
         self.assertSerializedEqual(True)
         self.assertSerializedEqual(False)
 
+    def test_serialize_field_with_lambda(self):
+        field = UnserializableField()
+
+        error = 'Cannot serialize function: lambda'
```

Comment:
Suggestion, write this as one string in full?

``` 
msg = (
    "Error during...."
    " ....."
)
```

Reply:
By using an f string it was easy to fit it all on one line without exceeding the character limit.